index.html

```diff
@@ -5,12 +5,53 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+      <img id="header-image" src="images/cupcake.png" alt="">
+      <nav id="nav-mobile">≡</nav>
+      <h2 id="header-title">The best cakes in town delivered to your door</h2>
```

Comment:
A common convention is to wrap the site logo/name in an anchor tag that links to the home page